### PR TITLE
Config update

### PR DESCRIPTION
**Pull request by girikuncoro**
Merge #27

### CODE REVIEW

File: app/api/config/main.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    require_apikey
+)
+
+
+app = Flask(__name__)
+enable_json_error(app)
+
+
+@app.route('/v1/config', methods=['GET'])
+@require_apikey
```
Comment (Ragil) [inline app/api/config/main.py]: This doesn't really make sense. We need an api key to set an api key :). Having login:admin in the yaml should secure this endpoint enough

File: app/api/config/main.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    if update == 'true':
+        admin_username = request.args.get('admin_username')
+        admin_apikey = request.args.get('admin_apikey')
+
+        if not admin_username or not admin_apikey:
+            abort(400, 'missing required parameters')
+
+        new = Config(admin_username=admin_username,
+                     admin_apikey=admin_apikey)
+        delete_config()
```
Comment (Ragil) [inline app/api/config/main.py]: maybe we shouldn't delete the config and simply update it if it exist. I can imagine this config growing with more properties and accidentally not setting some of them after deploying could be catastrophic.

**Comment by Ragil** [comment]
Having said that, looks good.

File: app/api/config/main.py (hunk 1 above)
Comment (girikuncoro) [inline app/api/config/main.py]: lol, I'll remove this

File: app/api/config/main.py (hunk 2 above)
Comment (girikuncoro) [inline app/api/config/main.py]: I just don't know how to update without knowing the key beforehand, any idea how to do that? or assume the admin knows the entity's key/id?

Comment (girikuncoro) [inline app/api/config/main.py]: oh I see, we can just do get first, and use the key to update the info back

**Pull request merged**